docs/misc/math-expr.md

```diff
@@ -154,6 +154,7 @@ See javadoc of java.lang.Math for detailed explanation for each function.
 |remainder|remainder(x, y) returns the remainder operation on two arguments as prescribed by the IEEE 754 standard|
 |rint|rint(x) returns value that is closest in value to x and is equal to a mathematical integer|
 |round|round(x, y) returns the value of the x rounded to the y decimal places. While x can be an integer or floating-point number, y must be an integer. The type of the return value is specified by that of x. y defaults to 0 if omitted. When y is negative, x is rounded on the left side of the y decimal points. If x is `NaN`, x returns 0. If x is infinity, x will be converted to the nearest finite double. |
+|safe_divide|safe_divide(x,y) returns the division of x by y if y is not equal to 0, returns 0 otherwise|
```

Comment:
it's better to put this new function along with the 'div' function above.

Reply:
What if either x or y is NULL?

Reply:
they appear to be alphabetically sorted right now, maybe we should consider leaving it where it is (though I don't have a strong opinion)

It is worth documenting the null handling behavior (especially if we change it to return null in SQL compatible null handling mode)